src/commands/main.mjs

```diff
@@ -205,6 +206,9 @@ export const createMainCommand = () => {
   createUnlinkCommand(program)
   createWatchCommand(program)
 
+  createLogsBuildCommand(program)
```

Comment:
Can we combine these into a generic `logs` command (like we do with `functions`) so that it shows up nice in the docs?